README.md

```diff
@@ -1,136 +1,138 @@
+| No, but yes. Native support is in development but you can still connect your phone to your car and start listening. | Finamp is Legal. Because Finamp requires a Jellyfin server for streaming Music the Admin is responsible for buying music from Artists. |
 
-We're currently in the process of redesigning Finamp to transform it into a modern, beautiful, and feature-rich music player made specifically for Jellyfin.  
-You can join the beta on [Google Play](https://play.google.com/store/apps/details?id=com.unicornsonlsd.finamp) and [Apple TestFlight](https://testflight.apple.com/join/UqHTQTSs), or download the latest beta APK from the [releases page](https://github.com/jmshrv/finamp/releases).  
-Please note that the beta is still work-in-progress, so the UI and functionality might be inconsistent or incomplete, and is not final. However, the beta is **fully functional and should be stable** enough for daily use.
+### Problems
+If you encounter any errors, issues, accessibility problems or the likes please feel free to look the the [issue tracker](https://github.com/jmshrv/finamp/issues) and open a new issue if there isnt one already.
 
----
 
-**Finamp** is a Jellyfin music player for Android and iOS. It's meant to give you a similar listening experience as traditional streaming services such as Spotify and Apple Music, but for the music that you already own. It's free, open-source software, just like Jellyfin itself.  
-Some of its features include:
+## Contributing
+### Code
+Just like any [FOSS software](https://en.wikipedia.org/wiki/Free_and_open-source_software) finamp also relies on your contributions!
+If you are interested you can consult the [Contribution Guidelines](https://github.com/jmshrv/finamp/blob/main/CONTRIBUTING.md) to get stated. Anything helps!
 
-- A welcoming user interface that looks modern & unique, but still familiar
-- Downloading files for offline listening and saving mobile data. Can use transcoded downloads to save even more space.
-- Transcoded streaming for saving mobile data
-- Beautiful dynamic colors that adapt to your media
-- Audio volume normalization ("ReplayGain") (Jellyfin 10.9+)
-- Lyrics (Jellyfin 10.9+)
-- Gapless playback
-- Android Auto support (coming soon™)
-- Full support for Jellyfin's "Playback Reporting" feature and plugin, letting you keep track of your listening activity
+### Translations
+You can also help out by translating Finamp using our [weblate page](https://hosted.weblate.org/engage/finamp/). Here is the current state of translations:
+<div align="center">
+<a href="https://hosted.weblate.org/engage/finamp/">
+  <img src="https://hosted.weblate.org/widget/finamp/finamp/horizontal-auto.svg" alt="Translation status" />
+</a>
+</div>
 
-***You need your own Jellyfin server to use Finamp. If you don't have one yet, take a look at [Jellyfin's website](https://jellyfin.org/) to learn more about it and how to set it up.***
+<details>
+<summary>
 
-## Getting Finamp
+## The Spice! (Screenshots)
```

Comment:
Let's stick with just "Screenshots". Maybe a few important screenshots could also be visible by default, and the remainder is collapsible?